modules/understanding-upgrade-channels.adoc

```diff
@@ -82,6 +83,30 @@ and pre-production and production environments that participate in connected cus
 You can use the `stable-{product-version}` channel to upgrade from a previous minor version of
 {product-title}.
 
+[discrete]
+== eus-4.6 channel
+
+In addition to the stable channel, certain minor versions of {product-title} offer an
+Extended Update Support[https://access.redhat.com/support/policy/updates/openshift#ocp4_phases] (EUS).
+These EUS versions extend the maintenance phase for customers with Premium Subscriptions
+to 14 months. {product-title} 4.6 is currently the only minor version with EUS.
+
+Although there is no difference between stable-4.6 and eus-4.6
+channels until {product-title} 4.6 transitions to EUS the phase, you can switch
+to the EUS channel as soon as it becomes available. When {product-title} 4.6
+transitions to the EUS phase of its lifecycle the stable-4.6 channel will
+no longer receive subsequent z-stream updates. After you upgrade a to a version that
+is exclusive to the EUS channel, that cluster will no longer be eligible for minor
```

Comment:
"upgrade a to" should be "upgrade a cluster to"?

Reply:
thanks, fixed that